test/blocks/tabs/tabs.test.js

```diff
@@ -35,4 +42,57 @@ describe('tabs', () => {
     await sendKeys({ press: 'Space' });
     expect(unSelectedBtn2.ariaSelected).to.equal('false');
   });
+
+  it('disables paddles when tabList is not scrollable', async () => {
+    setViewport({ width: DESKTOP_WIDTH, height: HEIGHT });
+    window.dispatchEvent(new Event('resize'));
+    await delay(200);
+    const leftPaddle = allTabs[0].querySelector('.paddle-left');
+    const rightPaddle = allTabs[0].querySelector('.paddle-right');
+
+    expect(leftPaddle.getAttribute('disabled')).to.equal('');
+    expect(rightPaddle.getAttribute('disabled')).to.equal('');
+  });
+
+  it('right paddle is visible when tabList is scrollable', async () => {
+    setViewport({ width: MOBILE_WIDTH, height: HEIGHT });
+    window.dispatchEvent(new Event('resize'));
+    const rightPaddle = allTabs[0].querySelector('.paddle-right');
+
+    expect(rightPaddle.getAttribute('disabled')).to.equal('');
```

Comment:
is this the correct matcher? Based on the descriptive text, I'd expect a match against visibility, like the display property or opacity. Also, seems like the right paddle should not be disabled when scrollable.

Reply:
Yes this is the correct matcher